src/Sylius/Bundle/AdminBundle/Resources/views/Product/_mainImage.html.twig

```diff
@@ -1,3 +1,3 @@
-{% set path = product.image is not null ? product.image.path|imagine_filter(filter|default('sylius_admin_product_thumbnail')) : 'http://placehold.it/50x50' %}
+{% set path = product.imageByCode('thumbnail') is not null ? product.imageByCode('thumbnail').path|imagine_filter(filter|default('sylius_admin_product_thumbnail')) : 'http://placehold.it/50x50' %}
```

Comment:
We should fallback to first image if there is no image with `thumbnail` code. Same for `main`